xds/internal/test/xds_client_integration_test.go

```diff
@@ -28,12 +28,19 @@ import (
 	"testing"
 
 	"google.golang.org/grpc"
+	_ "google.golang.org/grpc/balancer/rls"
```

Comment:
Please group blank imports in a separate group and add a one line comment for each as to why they are imported. I find that much more readable. Thanks.